### PR TITLE
Add Microsoft Teams actions plugin to microsite

### PR DESCRIPTION
**Pull request by grvpandey11**
## Hey, I just made a Pull Request!

Adds Microsoft Teams scaffolder actions plugin to the plugins list.

For use of Microsoft Teams logo:
- https://answers.microsoft.com/en-us/msteams/forum/all/use-of-logos/6f0a104b-ebf1-49c7-b76e-bcfea88baa5c

Please do let me know If I have to change the Icon.

Inspired by Andrew Hill Slack Plugin

#### :heavy_check_mark: Checklist

- [ ] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [x] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

**Comment by grvpandey11** [comment]
@benjdlambert The Verify Docs Quality check is failing on my name. Is there something I can do?

**Comment by adamdmharvey** [comment]
@grvpandey11 You can add your first name and surname here:

https://github.com/backstage/backstage/blob/master/.github/vale/Vocab/Backstage/accept.txt

Usually on pages with a lot of contributor names being added, those files would be excluded (like the adopters page), but since this is a main content page in the primary docs site, it's still spell checked.  Hope that helps!

**Comment by grvpandey11** [comment]
Thanks @adamdmharvey It certainly helped.

File: microsite/static/img/Ms-Teams.png
Comment (awanlin) [inline microsite/static/img/Ms-Teams.png:1]: We can't use this without consent from Microsoft: https://www.microsoft.com/en-us/legal/intellectualproperty/trademarks

Can you please change this?

**Review by awanlin — CHANGES_REQUESTED** [review]

File: microsite/static/img/Ms-Teams.png
Comment (grvpandey11) [reply microsite/static/img/Ms-Teams.png:1]: Fixed

Comment (awanlin) [reply microsite/static/img/Ms-Teams.png:1]: Sorry, I totally missed the comment about this in the PR. We generally err on the side of caution with things like this. Thanks for the understanding 👍

**Review by awanlin — APPROVED** [review]

**Comment by grvpandey11** [comment]
@benjdlambert Can you please have a look to this Pull Request.

**Review by Rugvip — APPROVED** [review]
Great, thanks! :+1: 

Thank you for helping out with reviews @adamdmharvey @awanlin :heart:

**Pull request merged**